src/components/calculator.js

```diff
@@ -1,52 +1,21 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import Key from './key';
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+import KeyBoard from './keyBoard';
+import Screen from './screen';
+import displayOnScreen from './functions/functions';
 
-function IntKeys() {
-  return (
-    <div className="intKeys-container">
-      <Key chart="AC" />
-      <Key chart="+/-" />
-      <Key chart="%" />
-      <Key chart="÷" />
-      <Key chart={7} />
-      <Key chart={8} />
-      <Key chart={9} />
-      <Key chart="*" />
-      <Key chart={4} />
-      <Key chart={5} />
-      <Key chart={6} />
-      <Key chart="-" />
-      <Key chart={1} />
-      <Key chart={2} />
-      <Key chart={3} />
-      <Key chart="+" />
-      <Key chart={0} />
-      <Key chart="." />
-      <Key chart="=" />
-    </div>
-  );
-}
-
-function Screen(props) {
-  const { value } = props;
-  return (
-    <input className="screen" type="text" value={value} />
-  );
-}
-
-Screen.defaultProps = {
-  value: '0',
-};
-
-Screen.propTypes = {
-  value: PropTypes.string,
-};
 function Calculator() {
+  const [calculateData, setCalculateData] = useState(
+    {
+      total: '',
+      next: '',
+      operator: '',
+    },
+  );
   return (
     <div>
-      <Screen value={0} />
-      <IntKeys />
+      <Screen value={displayOnScreen(calculateData)} />
+      <KeyBoard data={calculateData} setData={setCalculateData} calculate={calculate} />
     </div>
   );
 }
```

Comment:
![image](https://github.com/Grandi0z/Math_magicians/assets/93039359/0740d677-be6c-4ea7-a3f9-b2afeb3024ba)

- [ ] It would be nice if you start the name of your calculator.js component with a capital letter to make your project follow the requirements and React naming convention 👍